challenge/src/components/Pages/About.jsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react'
-import RandomImage from '../RandomImage'
+import JojoStand from '../JojoStand'
 
-const About = (props) => {
-    const page = props.currentpage
-    const pictureStyling = {
-        marginTop: "5%",
-        marginBottom: "5%",
-        marginRight: "5%",
-        marginLeft: "5%"
+class About extends React.Component {
+    constructor(props){
+      super(props);
+      this.state = {
+        standsVisibility: [
+          true,true,true,true
+        ],
+        length: 4
+      }
     }
-    return (
+    snap = () => {
+      var oldStands = this.state.standsVisibility.slice(0, this.state.length/2);
```

Comment:
this seems slightly more complicated than it has to be. perhaps old rand replaced can just the the whole array, and a clone of the whole array (you can do `[...oldStands]` to do a shallow clone of the array so react recognizes it as a different object.